src/tests/test_threecenter_dft.cc

```diff
@@ -51,8 +51,6 @@ Eigen::MatrixXd ReadMatrixFromString(const std::string& matrix) {
 
 BOOST_AUTO_TEST_CASE(small_basis) {
 
-  OPENMP::setMaxThreads(1);
```

Comment:
Why is it now possible to use multiple threads here?

Reply:
I am not sure why it was disabled before.